src/python/ensembl/io/genomio/metadata/prepare_genome.py

```diff
@@ -255,22 +258,31 @@ def prepare_genome_metadata(
     print_json(output_path, genome_data)
 
 
-class InputSchema(argschema.ArgSchema):
-    """Input arguments expected by the entry point of this module."""
-
-    json_file = argschema.fields.InputFile(
-        required=True, metadata={"description": "Genome metadata JSON file path"}
-    )
-    output_dir = argschema.fields.OutputDir(
-        required=False,
-        dump_default=".",
-        metadata={
-            "description": "Output folder for the updated genome metadata JSON file. By default, $PWD."
-        },
-    )
+def validate_input_path(parser: argparse.ArgumentParser, filepath: PathLike) -> PathLike:
+    """TODO"""
+    if Path(filepath).exists():
+        return filepath
+    parser.error(f"File '{filepath}' not found")
 
 
 def main() -> None:
     """Module's entry-point."""
-    mod = argschema.ArgSchemaParser(schema_type=InputSchema)
-    prepare_genome_metadata(mod.args["json_file"], mod.args["output_dir"])
+    parser = argparse.ArgumentParser(
+        description=("Expand the genome metadata with information about the provider, assembly and gene"
+                     "build version, and taxonomy."),
+    )
+    parser.add_argument(
+        "--json_file",
+        required=True,
+        type=lambda x: validate_input_path(parser, x),
+        help="Genome metadata JSON file path",
+    )
+    parser.add_argument(
+        "--output_dir",
+        default=Path().absolute(),
+        type=Path,
+        help="Output folder for the updated genome metadata JSON file (default: current working directory)",
```

Comment:
We can show the default automatically with an argument to the parser (https://stackoverflow.com/questions/12151306/argparse-way-to-include-default-values-in-help):
```
parser = argparse.ArgumentParser(
    # ... other options ...
    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```